src/components/Select/Option.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from 'react'
+
+interface OptionProps {
+  className?: string
+  value?: string | number
+  label: string
+}
+
+const Option: FC<OptionProps> = ({
+  className = 'select__option',
```

Comment:
dziwne bo takiej klasy nawet nie ma, ale widzę, że to jakis mega stary komponent